src/snapred/backend/data/GroceryService.py

```diff
@@ -222,11 +224,19 @@ def _createDiffcalDiagnosticWorkspaceFilename(self, item: GroceryListItem) -> st
             / (self._createDiffcalOutputWorkspaceName(item) + ext)
         )
 
+    def _createDiffcalTableFilepathFromWsName(
```

Comment:
I don't understand this.  I could do
`runNumber, useLiteMode, version = wsName.tokens('runNumber', 'useLiteMode', 'version')`
to get the required field values.  What is the purpose of the redundant arguments?

Reply:
**Also, and possibly more importantly, what happens when these argument values are in contradiction with each other?**